libs/components/src/popover/vl-popover.component.ts

```diff
@@ -0,0 +1,125 @@
+import { CSSResult, PropertyDeclarations, TemplateResult, html, PropertyValues } from 'lit';
+import { BaseLitElement } from '@domg-wc/common-utilities';
+import { resetStyle } from '@domg/govflanders-style/common';
+import { vlElementsStyle } from '@domg-wc/elements';
+import { customElement } from 'lit/decorators.js';
+import FloatingController from './vl-floating-ui.controller';
+import popoverUigStyle from './vl-popover.uig-css';
+import popoverLinkedListStyle from './vl-popover.linked-list.css';
+import type { Placement } from '@floating-ui/dom';
+
+@customElement('vl-popover')
+export class VlPopoverComponent extends BaseLitElement {
+    private popup!: FloatingController;
+
+    for = ''; // html id van het referentie element
+    trigger = 'click';
+    placement: Placement = 'bottom';
+    distance = 0; // afstand van referentie-element in px
+    minimumArrowDistance = 10; // minimum afstand in px van referentie-element wanneer er een pijl is
+    open = false;
+    hideArrow = false;
```

Comment:
Kan je accessors voorzien voor deze properties? Dan is het duidelijk dat ze public moeten zijn